common.gypi

```diff
@@ -468,38 +448,32 @@
             'cflags!': [ '-pthread' ],
             'ldflags!': [ '-pthread' ],
           }],
-          [ 'OS=="aix"', {
-            'variables': {'real_os_name': '<!(uname -s)',},
-            'conditions': [
-              [ 'target_arch=="ppc"', {
-                'ldflags': [
-                  '-Wl,-bmaxdata:0x60000000/dsa',
-                  '-Wl,-blibpath:/usr/lib:/lib:/opt/freeware/lib/pthread',
-                 ],
-              }],
-              [ 'target_arch=="ppc64"', {
-                'cflags': [ '-maix64' ],
-                'ldflags': [
-                  '-maix64',
```

Comment:
It'll probably be evident in the CI, but we need to pass `-maix64` to the linker as well as the compiler.

Reply:
Ack

Reply:
> It'll probably be evident in the CI, but we need to pass `-maix64` to the linker as well as the compiler.

FTR, this is a sample error message from `ld`:
```shell
ld: 0711-736 ERROR: Input file /home/iojs/build/workspace/node-test-commit-aix/nodes/aix61-ppc64/out/Release/obj.host/genccode/deps/icu-small/source/tools/genccode/genccode.o:
 	XCOFF64 object files are not allowed in 32-bit mode.
```